src/CAN/FakeCANBoard.cpp

```diff
@@ -160,11 +160,12 @@ int main() {
 			robot::types::datatime_t currTime = robot::types::dataclock::now();
 
 			// get y data: set point (target vel * time since set velocity call) + initial pos
-			double setPoint = (targetVel * util::durationToSec(currTime - startTime)) +
-							  initialMotorPos.getData();
+			[[maybe_unused]] double setPoint =
+				(targetVel * util::durationToSec(currTime - startTime)) +
+				initialMotorPos.getData();
```

Comment:
Instead of marking as unused, we should just print out `setPoint` and `motorPos`. It's an easy enough fix, can you add that?